ROMFS/px4fmu_common/mixers/vtol_tailsitter_duo.main.mix

```diff
@@ -29,11 +29,11 @@ Channel 5 connects to the right (starboard) elevon.
 Channel 6 connects to the left (port) elevon.
 
 M: 2
-O:        7500    7500    0 -10000  10000
+O:       10000   10000    0 -10000  10000
```

Comment:
```suggestion
```

In this case you can remove the `O:` line entirely as with your change it has no effect anymore.

Also bear in mind that this is very airframe specific. For certain, especially custom built frames, the maximum servo deflection will be too high and thus detrimental to performance, but I agree that this is not the normal scenario so it seems like a good improvement.

Reply:
good point, will do. 

I think if people want it limited then they need to create custom airframe configs/mixers. But the default should be full range.

Reply:
Yes, like I said I agree with your approach/logic.

Sounds good to me :)